server/svix-server/src/expired_message_cleaner.rs

```diff
@@ -0,0 +1,33 @@
+// SPDX-FileCopyrightText: © 2022 Svix Authors
+// SPDX-License-Identifier: MIT
+
+use crate::db::models::message;
+use crate::error::Result;
+use chrono::Utc;
+use sea_orm::{
+    entity::prelude::*, sea_query::Expr, DatabaseConnection, DbErr, EntityTrait, UpdateResult,
+};
+use tokio::time::sleep;
+
+/// Nullifies the payload column for expired messages
+pub async fn clean_expired_messages(
+    pool: &DatabaseConnection,
+) -> std::result::Result<UpdateResult, DbErr> {
+    message::Entity::update_many()
+        .col_expr(message::Column::Payload, Expr::value(Value::Json(None)))
```

Comment:
```suggestion
        .col_expr(message::Column::Payload, Expr::value(None))
```

Reply:
Unfortunately this does not compile:
```
type inside `async fn` body must be known in this context
cannot infer type for type parameter `T` declared on the enum `Option`
```

Reply:
https://www.sea-ql.org/SeaORM/docs/basic-crud/update/#update-many

Reply:
It's actually correct and the docs are wrong.